pkg/controller/constrainttemplate/constrainttemplate_controller_test.go

```diff
@@ -739,11 +777,27 @@ func TestReconcile(t *testing.T) {
 		err = retry.OnError(testutils.ConstantRetry, func(_ error) bool {
 			return true
 		}, func() error {
+			ct := &v1beta1.ConstraintTemplate{}
+			if err := c.Get(ctx, types.NamespacedName{Name: denyall + strings.ToLower(suffix)}, ct); err != nil {
+				return err
+			}
+			timestamp := ct.GetAnnotations()[constraint.BlockVAPBGenerationUntilAnnotation]
+			if timestamp == "" {
+				return fmt.Errorf("expected %s annotations on CT", constraint.BlockVAPBGenerationUntilAnnotation)
+			}
 			// check if vapbinding resource exists now
 			vapBinding := &admissionregistrationv1.ValidatingAdmissionPolicyBinding{}
 			if err := c.Get(ctx, types.NamespacedName{Name: vapBindingName}, vapBinding); err != nil {
 				return err
 			}
+			// blockTime, err := time.Parse(time.RFC3339, timestamp)
+			// if err != nil {
+			// 	return err
+			// }
+			// currentTime := time.Now()
+			// if currentTime.Before(blockTime) {
+			// 	t.Fatal("VAPBinding should not be created before the timestamp", currentTime, blockTime)
+			// }
```

Comment:
why is this commented out?

Reply:
removed the commented block. Updated the tests as well.